views/aggregate-block-metadata.json

```diff
@@ -3587,7 +3587,7 @@
       "databricks-credentials": {
         "name": "Databricks Credentials",
         "slug": "databricks-credentials",
-        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/ff9a2573c23954bedd27b0f420465a55b1a99dfd-250x250.png",
+        "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/5GTHI1PH2dTiantfps6Fnc/1c750fab7f4c14ea1b93a62b9fea6a94/databricks_logo_icon_170295.png?h=250",
```

Comment:
@desertaxle is this one right? the rest of this lgtm

Reply:
No, let me revert that change